### PR TITLE
Parental leave - Usage slider fixes

### PR DESCRIPTION
**Pull request by madebynoam**
1. Don't allow click past min number
2. Use `left` instead of `transformX` for remainder bar position, so that the remaining width is always correct - fixes a big where it was overflowing visually on the right.

### CODE REVIEW

File: libs/application/form/src/components/fields/ParentalLeave/Usage/components/Slider.tsx [hunk 1]
```diff
@@ -152,14 +154,21 @@ const Slider = ({
     transform: `translateX(${dragX.current == null ? x : dragX.current}px)`,
     transition: isDragging ? 'none' : '',
   }
+  const remiainderStyle = {
```
Comment (ArniFreyrSnorrason) [inline libs/application/form/src/components/fields/ParentalLeave/Usage/components/Slider.tsx:157]: remainderStyle

**Review by ArniFreyrSnorrason — DISMISSED** [review]
LGTM

File: libs/application/form/src/components/fields/ParentalLeave/Usage/components/Slider.tsx (hunk 1 above)
Comment (madebynoam) [reply libs/application/form/src/components/fields/ParentalLeave/Usage/components/Slider.tsx]: oops good catch :) fixed now.

**Comment by madebynoam** [comment]
Awesome thanks for the review!

**Comment by madebynoam** [comment]
Added one more option while looking at the file - to snap to the next slider step. We can decide if we want the smoother transition of the remainder bar or the snap. I think snap might be better for usability.

slider-snap

**Review by ArniFreyrSnorrason — APPROVED** [review]

**Pull request merged**